### PR TITLE
Added password login support and two more changes

### PR DESCRIPTION
**Pull request by SwetyCore**
+ Thanks for [this](https://github.com/Jx2f/mhypbase) project, we can decrypt the login password through SigningKey Pem on Windows platform now.
+ [~~Handled SetUpAvatarTeamReq packet~~](https://github.com/WeedwackerPS/Weedwacker/pull/3/commits/6bc1a78830dea750d8aad85d68cf8827f828032f) 
+ [Added Unhandled packet log notice color](https://github.com/WeedwackerPS/Weedwacker/pull/3/commits/26ce36f6121391daa84bedbf800304718057abb8)
+ Updated README.me

### CODE REVIEW

File: src/GameServer/Packet/Recv/HandleSetUpAvatarTeamReq.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+namespace Weedwacker.GameServer.Packet.Recv
+{
+    [OpCode((ushort)OpCode.SetUpAvatarTeamReq)]
+    internal class HandleSetUpAvatarTeamReq : BaseHandler
+    {
+        public override async Task HandleAsync(Connection session, byte[] header, byte[] payload)
+        {
+            SetUpAvatarTeamReq proto = SetUpAvatarTeamReq.Parser.ParseFrom(payload);
+            
+            await session.Player.TeamManager.SetupAvatarTeamAsync((int)proto.TeamId, proto.AvatarTeamGuidList.Select(x=>(long)x).ToList());
```
Comment (akatatsu27) [inline src/GameServer/Packet/Recv/HandleSetUpAvatarTeamReq.cs:18]: This is irrelevant to this PR. It doesn't send the rsp packet, and I get stuck in infinite loading. This should be part of a separate PR.
Also, changing SetupAvatarTeamAsync to take `IList<ulong>` instead of `List<long>` as an argument would remove the need for the proto.AvatarTeamGuidList.Select lambda

File: src/Shared/Utils/Configuration/ConfigFile.cs [hunk 2]
```diff
@@ -6,7 +6,7 @@ public class ConfigFile
     {
         public StructureJson structure = new();
         public Language language = new();
-        public int version = 1;
+        public int version = 2;
```
Comment (akatatsu27) [inline src/Shared/Utils/Configuration/ConfigFile.cs:9]: Now that I think about it, a Major.Minor versioning scheme seems more reasonable.
```suggestion
        public string version = "1.1";
```

File: src/WebServer/Authentication/DefaultAuthenticators.cs [hunk 3]
```diff
@@ -22,6 +25,12 @@ public async Task<LoginResultJson> Authenticate(AuthenticationRequest request)
             string address = request.Context.Connection.RemoteIpAddress.ToString();
             string responseMessage = "Username not found.";
             string loggerMessage = "";
+            string decryptedPasswordMd5 = "";
+
+            if (WebServer.Configuration.Server.Account.UsePassword)
+            {
+                decryptedPasswordMd5 = Crypto.GetPasswordHash(requestData.password);
```
Comment (akatatsu27) [inline src/WebServer/Authentication/DefaultAuthenticators.cs:32]: I tried numerous passwords, but GetPasswordHash always throws the exception "The length of the data to decrypt is not valid for the size of this key.", and returns "".
I've also modified the command handler to allow variable length commands to allow an optional password field in the createaccount command

File: src/Shared/Utils/Logger.cs [hunk 4]
```diff
@@ -20,6 +20,15 @@ public static void DebugWriteLine(string message)
             Console.WriteLine(ParseMessage(message));
 #endif
         }
+
+        public static void DebugWriteWarningLine(string message)
```
Comment (akatatsu27) [inline src/Shared/Utils/Logger.cs:24]: I like this addition. Please mention it in the title and/or the description

**Review by akatatsu27 — CHANGES_REQUESTED** [review]

File: src/GameServer/Packet/Recv/HandleSetUpAvatarTeamReq.cs (hunk 1 above)
Comment (SwetyCore) [reply src/GameServer/Packet/Recv/HandleSetUpAvatarTeamReq.cs]: Removed

**Review by akatatsu27 — APPROVED** [review]
